protonvpn_cli/utils.py

```diff
@@ -29,21 +33,26 @@ def call_api(url, json_format=True):
 
     logger.debug("Initiating API Call: {0}".format(url))
 
+    # For manual error handling, such as in wait_for_network()
+    if not handle_errors:
+        response = requests.get(url, headers=headers)
+        return response
+
     try:
         response = requests.get(url, headers=headers)
     except (requests.exceptions.ConnectionError,
             requests.exceptions.ConnectTimeout):
         print(
-            "[!] There was an error connecting to the ProtonMail API.\n"
+            "[!] There was an error connecting to the ProtonVPN API.\n"
             "[!] Please make sure your connection is working properly!"
         )
-        logger.debug("Error connecting to ProtonMail API")
+        logger.debug("Error connecting to ProtonVPN API")
         sys.exit(1)
     try:
         response.raise_for_status()
     except requests.exceptions.HTTPError:
         print(
-            "[!] There was an error with accessing the ProtonMail API.\n"
+            "[!] There was an error with accessing the ProtonVPN API.\n"
```

Comment:
In fact now that I am realizing: all the errors should be logged as error (on `stderr`)